### DISCUSSION TITLE
v0.6.0

### DISCUSSION THREAD

**Discussion by gilwong00** [connectrpc/connect-es#398] [Announcements]
This release includes the following:

## 🚨 Breaking Changes 🚨 
* #370 - The Connect protocol now specifies that a header be set specifying the version of Connect in use.  The header name is `Connect-Protocol-Version` and Connect-Web has been updated to send this header in its requests.  As a result, users will need to make sure their CORS settings allowlist this new header, or all _Connect_ requests will fail with this release.  

   For more information, see this PR:  https://github.com/bufbuild/connect-go/pull/416

## Enhancements
* Add Connect Protocol Version Header by @gilwong00 in #370

<hr /><em>This discussion was created from the release <a href='https://github.com/bufbuild/connect-web/releases/tag/v0.6.0'>v0.6.0</a>.</em>

**Comment by nii236** [comment]
This took me WAY too long to figure out. Is there an issue tracking the updating of documentation (wiki, guides, tutorials) to reflect this change?

**Comment by smaye81** [comment]
There isn't, but if you experienced this difficulty then others probably have too.  We should add something to our FAQ.  Can you elaborate on what specifically was giving you trouble?

**Reply by nii236** [reply]
Backend developers (Go, in my case) explicitly need to allow headers `Connect-Protocol-Version` into their CORS middleware. This wasn't immediately clear and I had to dig through pull requests (and consequently, this thread) to figure it out.

I recommend perhaps adding this in the Common Errors section (https://connect.build/docs/go/common-errors).

**Reply by smaye81** [reply]
Sounds good.  Thanks for calling this out.  We'll get it added.

**Reply by smaye81** [reply]
Added a Common Errors page to Connect-Web docs here:  https://connect.build/docs/web/common-errors
